tomlkit/items.py

```diff
@@ -1574,7 +1574,7 @@ def __copy__(self) -> "Table":
             self.display_name,
         )
 
-    def append(self, key, _item):
+    def append(self, key: Union[Key, str], _item: Any) -> "Table":
```

Comment:
It seems key can be `None`, just add `None` to the Union

Reply:
Sure! So, these methods will have the same types as the `append()` method of the `Container` class, right?

Reply:
@frostming , added `None` to the `append()` and `raw_append()` methods and fixed the linting errors. Can you check the MR again, please?